model/Movie.js

```diff
@@ -0,0 +1,43 @@
+import mongoose from "mongoose";
+
+const { Schema, model } = mongoose;
+
+const movieSchema = new Schema({
+  show_id: {
+    type: Number,
+  },
```

Comment:
there's no need to pass an id to your schema, remove `id` as Mongoose automatically assigns a unique `_id` field to each document, so you don't need to define an `id` field  👍